src/app/components/shell/account-popover/account-popover.vue

```diff
@@ -30,50 +33,50 @@
 					</router-link>
 				</div>
 
-				<div class="account-popover-separator"></div>
+				<div class="account-popover-separator" />
 
 				<div class="list-group-dark">
 					<router-link
+						v-app-track-event="`account-popover:library`"
 						class="list-group-item"
 						:to="{ name: GJ_IS_CLIENT ? 'library.installed' : 'library.overview' }"
-						v-app-track-event="`account-popover:library`"
 					>
 						<translate>Game Library</translate>
 					</router-link>
 					<router-link
+						v-app-track-event="`account-popover:account`"
 						class="list-group-item offline-disable"
 						:to="{
 							name: Screen.isXs ? 'dash.account-mobile-nav' : 'dash.account.edit',
 						}"
-						v-app-track-event="`account-popover:account`"
 					>
 						<translate>Edit Account</translate>
 					</router-link>
 					<router-link
+						v-app-track-event="`account-popover:stickers`"
 						class="list-group-item offline-disable"
 						:to="{ name: 'dash.stickers.overview' }"
-						v-app-track-event="`account-popover:stickers`"
 					>
 						<translate>Stickers</translate>
-						<span v-if="shouldShowNewStickers" class="-new-tag -new-tag-list"></span>
```

Comment:
Also can't self-close spans.
Reference: https://html.spec.whatwg.org/multipage/syntax.html#start-tags (specifically 6.)
Valid self-closing elements are the "void elements" in this specification: https://html.spec.whatwg.org/multipage/syntax.html#void-elements